src/screens/ExpandedAssetSheet.js

```diff
@@ -50,7 +51,9 @@ export default function ExpandedAssetSheet(props) {
   // We want to revalidate (ie. refresh OpenSea metadata) collectibles
   // to ensure the user can get the latest metadata of their collectible.
   const selectedAsset = useAsset(params.asset, {
-    revalidateCollectibleInBackground: true,
+    revalidateCollectibleInBackground: isUnknownOpenSeaENSDescription(
```

Comment:
Do we not want to do it for all NFTs? I don't see a downside since this revalidation is happening in the background.